static/js/components/LearnerInfoCard.js

```diff
@@ -18,50 +22,85 @@ export default class LearnerInfoCard extends React.Component {
     profile: Profile,
     toggleShowPersonalDialog: () => void,
     toggleShowAboutMeDialog: () => void,
+    openLearnerEmailComposer: () => void,
   };
 
-  email = (email: string): React$Element<*> => (
-    <span className="profile-email">{email}</span>
-  );
+  ownProfilePage: boolean;
+
+  isOwnProfilePage = (): boolean => {
+    if (R.isNil(this.ownProfilePage)) {
+      this.ownProfilePage = isProfileOfLoggedinUser(this.props.profile);
+    }
+    return this.ownProfilePage;
+  };
```

Comment:
OK, just fixed this to use repeated calculations. I know there has to be a solution that cleanly eliminates the need for recalculating this value (and others I've seen around the codebase), but I don't really know enough about the React lifecycle yet, and as you said it's very cheap in this case